oracle/src/events/processCollectedSignatures/index.js

```diff
@@ -46,6 +49,37 @@ function processCollectedSignaturesBuilder(config) {
         logger.info(`Processing CollectedSignatures ${colSignature.transactionHash}`)
         const message = await homeBridge.methods.message(messageHash).call()
 
+        if (ORACLE_HOME_TO_FOREIGN_ALLOWANCE_LIST || ORACLE_HOME_TO_FOREIGN_BLOCK_LIST) {
+          const parsedMessage = parseMessage(message)
+          const recipient = parsedMessage.recipient.toLowerCase()
+          const originalTxHash = parsedMessage.txHash
+
+          if (ORACLE_HOME_TO_FOREIGN_ALLOWANCE_LIST) {
+            const allowanceList = await readAccessListFile(ORACLE_HOME_TO_FOREIGN_ALLOWANCE_LIST)
+            if (allowanceList.indexOf(recipient) === -1) {
+              const sender = (await web3Home.eth.getTransaction(originalTxHash)).from.toLowerCase()
```

Comment:
Can we add a debug message that the transaction is requested to get withdrawal request sender? The same of the block list...